client/src/component/Dashboard.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import CodeWarsCard from "/CodeWarsCard";
+
+const Dashboard = () => {
```

Comment:
Ageine: The export keyword is used to export the Dashboard component as a named export. This means that other files can import and use the Dashboard component by explicitly referencing its name.export, change the line into: const Dashboard = () => {